### PR TITLE
Implement armour stand marker data

### PR DESCRIPTION
**Pull request by kashike**
[API](https://github.com/SpongePowered/SpongeAPI/pull/1100) | [**Common**](https://github.com/SpongePowered/SpongeCommon/pull/533)

### CODE REVIEW

File: src/main/java/org/spongepowered/common/command/SpongeCommandManager.java [hunk 1]
```diff
@@ -143,6 +144,7 @@ public SpongeCommandManager(Logger logger, Disambiguator disambiguator) {
 
             if (mapping.isPresent()) {
                 this.owners.put(container, mapping.get());
+                Sponge.getServiceManager().provide(PermissionService.class).ifPresent(callable::registerPermissions);
```
Comment (simon816) [inline src/main/java/org/spongepowered/common/command/SpongeCommandManager.java]: What's this?

Comment (kashike) [inline src/main/java/org/spongepowered/common/command/SpongeCommandManager.java]: seems something really old snuck in here somehow

File: src/main/java/org/spongepowered/common/data/processor/data/entity/MarkerDataProcessor.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    @Override
+    protected Optional<Boolean> getVal(EntityArmorStand container) {
+        return OptBool.of(container.hasMarker());
+    }
+
+    @Override
+    protected ImmutableValue<Boolean> constructImmutableValue(Boolean value) {
+        return new ImmutableSpongeValue<>(this.key, false, value);
```
Comment (simon816) [inline src/main/java/org/spongepowered/common/data/processor/data/entity/MarkerDataProcessor.java]: Use value cache

File: src/main/java/org/spongepowered/common/data/processor/data/entity/MarkerDataProcessor.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    @Override
+    protected Value<Boolean> constructValue(Boolean actualValue) {
+        return new SpongeValue<>(this.key, false, actualValue);
```
Comment (simon816) [inline src/main/java/org/spongepowered/common/data/processor/data/entity/MarkerDataProcessor.java]: Use value cache

**Comment by gabizou** [comment]
Closing as the API PR is asked to be redone as data proper.

**Pull request closed without merging**